_data/simple-icons.json

```diff
@@ -6431,6 +6421,12 @@
             "source": "https://openwrt.org/docs/guide-graphic-designer/openwrt-logo",
             "guidelines": "https://openwrt.org/docs/guide-graphic-designer/openwrt-logo"
         },
+        {
+            "title": "OpenSea",
+            "hex": "2081E2",
```

Comment:
Being completely unfamiliar with the brand myself and for future reference, why did you choose this color over any of the others listed in [the source](https://docs.opensea.io/docs/logos)?

Reply:
They primarily use this color over their darker color on their logo and website.